lib/verk/queue_stats_counters.ex

```diff
@@ -0,0 +1,82 @@
+defmodule Verk.QueueStatsCounters do
+  @moduledoc """
+  This module is responsible for abstracting the logic of keeping counters for
+  each queue.
+  """
+
+  @table :queue_stats
+  @previous_table :previous_queue_stats
+  @persist_interval 1000
+
+  @doc """
+  Initializes the ets tables for the queue stats.
+  """
+  def init do
```

Comment:
let's add a @spec?

Always return `:ok` so the spec is 

`@spec init :: :ok` /shrug